.gitignore

```
@@ -4,3 +4,5 @@ node_modules
 build
 .cache
 dev
+dist/
```

```suggestion
dist
```
Consistency with others listed here